cmd/srcd/cmd/components.go

```diff
@@ -145,10 +132,71 @@ var componentsInstallCmd = &cobra.Command{
 	},
 }
 
+// componentsStartCmd represents the components start command
+var componentsStartCmd = &cobra.Command{
+	Use:   "start [component]",
+	Short: "Start source{d} component",
+	Args:  cobra.MinimumNArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		client, err := daemon.Client()
+		if err != nil {
+			return humanizef(err, "could not get daemon client")
+		}
+
+		cmps, err := components.List(context.Background(), false)
+		if err != nil {
+			return humanizef(err, "could not list images")
+		}
+
+		for _, arg := range args {
+			c, err := getComponent(arg, cmps)
+			if err != nil {
+				return err
+			}
+
+			ctx := context.Background()
+			started := logAfterTimeoutWithServerLogs("this is taking a while, "+
+				"it might take a few more minutes while we install all the required images",
+				5*time.Second)
+			_, err = client.StartComponent(ctx, &api.StartComponentRequest{
+				Name: c.Name,
+			})
+			started()
+			if err != nil {
+				return humanizef(err, "could not start %s", c.Name)
```

Comment:
Maybe we could build the list of errors so that we try to start each component that has been passed.

Reply:
not sure we really need it

Reply:
I don't have a strong opinion on this, so it's up to you.